lib/workers/repository/process/extract-update.ts

```diff
@@ -122,6 +124,7 @@ export async function lookup(
   packageFiles: Record<string, PackageFile[]>
 ): Promise<ExtractResult> {
   await fetchUpdates(config, packageFiles);
+  ensureGithubToken(packageFiles);
```

Comment:
Warning after the fetch phase

Reply:
does this make sense? as the datasources are already called here.

Reply:
`sourceUrl` can either be set on the extract phase or only after updates fetching. Example: extracted datasource is maven, but `sourceUrl` points to github.

Reply:
yes, but then the changlog retrival will already warn about missing token, `sourceUrl` is then only used for changelog and linking 😉

Reply:
Okay, I see now